apps/web/app/(core)/metadata.ts

```diff
@@ -1,21 +1,22 @@
 import {Metadata} from "next";
-import {getBrandText} from "@/src/utils/brandName";
+import {getBrandText, isRebrandEnabled} from "@/src/utils/brandName";
 
 const brandText = getBrandText();
 
 export const metadata: Metadata = {
   title: `${brandText.name} DEX - The Liquidity Hub on Fuel`,
   description: `Trade tokens and provide liquidity on ${brandText.dex}, the most efficient AMM on Fuel Network.`,
   icons: {
-    icon: "/favicon.ico",
+    icon: isRebrandEnabled() ? "/rebrand-favicon.png" : "/favicon.ico",
```

Comment:
you shouldnt need to feature flag, i think with the combo of code rabit and that this doesnt need feature flagging you should just clean up the line